etl/steps/data/meadow/smoking/2024-05-30/cigarette_sales.py

```diff
@@ -0,0 +1,265 @@
+    "Yugoslavia": {"file_name": "ISS-Yugoslavia_160705.xls", "sheet_name": "Table2.1_YUGF"},
+    "Croatia": {"file_name": "ISS-Yugoslavia_160705.xls", "sheet_name": "Table2.2_CROA"},
+    "Slovenia": {"file_name": "ISS-Yugoslavia_160705.xls", "sheet_name": "Table2.3_SLON"},
+    # special case: Bosnia, Macedonia and Yugoslavia are in one sheet with a different format
+    "Bosnia & Herzegovina": {"file_name": "ISS-Yugoslavia_160705.xls", "sheet_name": "Table2.4_YUGF"},
+    "Macedonia": {"file_name": "ISS-Yugoslavia_160705.xls", "sheet_name": "Table2.4_YUGF"},
+    "Federal Republic of Yugoslavia": {"file_name": "ISS-Yugoslavia_160705.xls", "sheet_name": "Table2.4_YUGF"},
+}
+
+SPECIAL_CASES = [
+    "Germany",
+    "USSR and fSU",
+    "Armenia",
+    "Azerbaijan",
+    "Belarus",
+    "Georgia",
+    "Kazakhstan",
+    "Kyrgyzstan",
+    "Moldova",
+    "Russia",
+    "Tajikistan",
+    "Turkmenistan",
+    "Ukraine",
+    "Uzbekistan",
+    "Bosnia & Herzegovina",
+    "Macedonia",
+    "Federal Republic of Yugoslavia",
+]
+
+FORMAT_SPECIAL_CASES = {
+    "Armenia": {"header_row": 9, "cols": "A,B,C", "num_rows": 13},
+    "Azerbaijan": {"header_row": 9, "cols": "A,E,F", "num_rows": 13},
+    "Belarus": {"header_row": 9, "cols": "A,H,I", "num_rows": 13},
+    "Georgia": {"header_row": 9, "cols": "A,K,L", "num_rows": 14},
+    "Kazakhstan": {"header_row": 25, "cols": "A,B,C", "num_rows": 15},
+    "Kyrgyzstan": {"header_row": 25, "cols": "A,E,F", "num_rows": 13},
+    "Moldova": {"header_row": 25, "cols": "A,H,I", "num_rows": 14},
+    "Russia": {"header_row": 25, "cols": "A,K,L", "num_rows": 19},
+    "Tajikistan": {"header_row": 46, "cols": "A,B,C", "num_rows": 13},
+    "Turkmenistan": {"header_row": 46, "cols": "A,E,F", "num_rows": 13},
+    "Ukraine": {"header_row": 46, "cols": "A,H,I", "num_rows": 18},
+    "Uzbekistan": {"header_row": 46, "cols": "A,K,L", "num_rows": 15},
+    "Bosnia & Herzegovina": {"header_row": 9, "cols": "A,B,C", "num_rows": 3},
+    "Macedonia": {"header_row": 14, "cols": "A,B,C", "num_rows": 4},
+    "Federal Republic of Yugoslavia": {"header_row": 20, "cols": "A,B,C", "num_rows": 9},
+}
+
+NEW_COLUMNS = [
+    "year",
+    "manufactured_cigarettes_millions",
+    "manufactured_cigarettes_per_adult_per_day",
+    "handrolled_cigarettes_millions",
+    "handrolled_cigarettes_per_adult_per_day",
+    "total_cigarettes_millions",
+    "total_cigarettes_per_adult_per_day",
+    "all_tobacco_products_tonnes",
+    "all_tobacco_products_grams_per_adult_per_day",
+]
+
+ORIGINAL_COLUMNS = [
+    "Year",
+    "Manufactured cigarettes",
+    "Unnamed: 2",
+    "Unnamed: 3",
+    "Hand-rolled cigarettes",
+    "Unnamed: 5",
+    "Unnamed: 6",
+    "Total cigarettes",
+    "Unnamed: 8",
+    "Unnamed: 9",
+    "All tobacco products",
+    "Unnamed: 11",
+    "Unnamed: 12",
+]
+
+
+def is_not_string(value):
+    return not isinstance(value, str)
```

Comment:
I don't think this function is necessary. You are creating more code, and simply having `not isinstance(value, str)` is clear enough.